larq_compute_engine/core/types.h

```diff
@@ -17,6 +19,12 @@ namespace core {
 #define LCE_UNLIKELY(condition) (condition)
 #endif
 
+#if defined(__GNUC__)
+#define FORCE_INLINE __attribute__((always_inline)) inline
+#else
+#define FORCE_INLINE inline
+#endif
```

Comment:
Maybe it is my search failing, but I don't think you use `FORCE_INLINE` somewhere?

Reply:
Indeed it is not used anywhere at the moment. I mainly added it to sync `larq_compute_engine/core/types.h` with our internal fork to reduce confusing for us in the future. But I am happy to remove it, if you prefer.

Reply:
Yea i'd opt to keep public and private fork as much in sync as possible.